dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -310,17 +311,19 @@ public void saveStep(WorkflowStep step) throws DotDataException, AlreadyExistExc
 	@WrapInTransaction
 	public void deleteStep(final WorkflowStep step) throws DotDataException {
 
+		Savepoint savepoint = null;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "savepoint" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)

Reply:
DONE